Utilities/sandbox/parse-manifest.sb

```diff
@@ -0,0 +1,19 @@
+(version 1)
+
+; Deny everything by default.
+(deny default)
+
+; Allow reading all files.
+(allow file-read*)
```

Comment:
Technically, I think the manifest should use only the standard library (from the toolchain), right?  i.e. it should not really need anything from the SDK.